docker-compose.yaml

```diff
@@ -147,8 +147,10 @@ services:
       - WORKER_ENVIRONMENT=${WORKER_ENVIRONMENT}
       - WORKSPACE_ROOT=${WORKSPACE_ROOT}
       - GITHUB_STORE_BRANCH=${GITHUB_STORE_BRANCH}
+      - MICRONAUT_ENVIRONMENTS=${WORKERS_MICRONAUT_ENVIRONMENTS}
     ports:
       - 8001
+      - 8042
```

Comment:
Shouldn't this match the new port you added above `:8080`?

Reply:
Yes, that's a mistake.

Reply:
Done